guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.*
```

Comment:
I would italicize the publication date / author text rather than the short summary of the paper. Italics are less emphatic and the summary is the primary information here.

Reply:
@johnsBeharry and I talked on Slack about formatting, but we thought we'd keep that for a separate PR so we can wrap this one up.